src/compile/common.ts

```diff
@@ -103,7 +103,9 @@ export function formatMixins(model: Model, channel: Channel, format: string) {
   let def: any = {};
 
   if (fieldDef.type === TEMPORAL) {
-    def.formatType = 'time';
+    if ((!fieldDef.timeUnit) || !containsTimeUnit(fieldDef.timeUnit, TimeUnit.QUARTER)) {
```

Comment:
Can you comment why if there is no `timeUnit` or if the `timeUnit` is quarter we don't assign `formatType` here?